http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200427_create_persistent_disk_table.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="rtitle" id="persistent-disk-table">
+        <createTable tableName="PERSISTENT_DISK">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="googleProject" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="zone" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="googleId" type="VARCHAR(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="samResourceId" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="status" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="creator" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="createdDate" type="TIMESTAMP(6)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="destroyedDate" type="TIMESTAMP(6)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="dateAccessed" type="TIMESTAMP(6)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="sizeGb" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="type" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="blockSizeBytes" type="INT">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addUniqueConstraint columnNames="googleProject, name, destroyedDate" constraintName="IDX_PERSISTENT_DISK_UNIQUE" tableName="PERSISTENT_DISK"/>
+        <createIndex indexName="IDX_PERSISTENT_DISK_STATUS" tableName="PERSISTENT_DISK">
+            <column name="status"/>
+        </createIndex>
+    </changeSet>
+    <changeSet logicalFilePath="leonardo" author="rtitle" id="persistent-disk-label-table">
+        <createTable tableName="PERSISTENT_DISK_LABEL">
```

Comment:
Thinking more, I think I'm going to try and repurpose the existing `LABEL` table instead of adding a new `PERSISTENT_DISK_LABEL`

Reply:
➕ I was starting to write a comment along the lines of this

Reply:
If disks can get errored out, I wonder if we should do the same for the `CLUSTER_ERROR` table

Reply:
That's an excellent question… will think about it but probably punt to a separate PR if so.